### DISCUSSION TITLE
architecture/MicroService

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#206] [General]
# architecture/MicroService

构建大规模高可用的分布式系统

https://thebyte.com.cn/architecture/MicroService.html

**Comment by Vampx** [comment]
Spring Cloud Kubernetes 组件有了解过吗

**Reply by SaintKayLuk** [reply]
哦？看样子又有新的东西要学了

**Reply by lzfgit-byte** [reply]
新玩具
